js/modules/swiperCommentsFunctionality.js

```diff
@@ -59,16 +60,20 @@
 		}
 	})
 
-	const mqOver1920 = window.matchMedia("(min-width: 1920px)");
-	mqOver1920.addEventListener("change", function(){
+	const mqlOver1920 = window.matchMedia("(min-width: 1920px)");
```

Comment:
Are you sure that there must be any difference in logic between, for example,  screen 1900 px and screen 1930px ?
maybe you should use one mq for min-width 1440 or something like this?

Reply:
Maybe you are right but it was in Figma. If you want, I'll redo it for another resolution. Waiting for an answer...

Reply:
I looked at the design, I agree, design is shit. But anyway you have min width 1920 and max width 1919. Use only min with 1920 and `!mediaQuire.matches` for screens less then 1920.

Reply:
Done